### PR TITLE
Add code gen for php guzzle

### LINKED ISSUE #143: Validate PHP (guzzle) CodeGen & add all test cases
**Tell us about the task you want to perform and are unable to do so because the feature is not available**

Add CodeGen for PHP
Library - https://github.com/guzzle/guzzle

More info here - https://github.com/foss42/apidash/discussions/80

### PR DESCRIPTION
**Pull request by ashutosh7i**
## PR Description

Wrote all tests for PHP guzzle codegen module, modified the code, bugfixes, all tests are passing and manually tested for all cases.

## Related Issues

- Related Issue #143 
- Closes #143

### Checklist
- [x] I have gone through the [contributing guide](https://github.com/foss42/apidash/blob/main/CONTRIBUTING.md)
- [x] I have updated my branch and synced it with project `main` branch before making this PR
- [x] I have run the tests (`flutter test`) and all tests are passing

## Added/updated tests?
_We encourage you to add relevant test cases._

- [x] Yes
- [ ] No, and this is why: _please replace this line with details on why tests have not been included_

### CODE REVIEW

File: lib/codegen/php/guzzle.dart [hunk 1]
```diff
@@ -117,11 +126,11 @@ echo \$res->getBody();
...
-      if (requestModel.hasFormData && requestModel.formDataMapList.isNotEmpty) {
-        result += templateBody.render({
-          "body": "new MultipartStream(\$multipart)",
-        });
-      }
+      // if (requestModel.hasFormData && requestModel.formDataMapList.isNotEmpty) {
+      //   result += templateBody.render({
+      //     "body": "new MultipartStream(\$multipart)",
+      //   });
+      // }
```
Comment (animator) [inline lib/codegen/php/guzzle.dart:133]: Remove comments.

**Comment by animator** [comment]
comment above 👆

File: lib/codegen/php/guzzle.dart (hunk 1 above)
Comment (ashutosh7i) [reply lib/codegen/php/guzzle.dart]: Done✅, removed comments.

**Comment by ashitaprasad** [comment]
@ashutosh7i Provide all instructions to run the code locally in the comment below 👇

**Comment by ashutosh7i** [comment]
so this PR is an update to merged PR [#256 ](https://github.com/foss42/apidash/pull/256)
in that PR tests were not included.

While writing tests for the same i found some bugs in the merged code, so this PR contains updated code with bugfixes.

To run the tests-
in terminal, from the root directory of the project, run
``` flutter test test/codegen/php_guzzle_codegen_test.dart ```

**Comment by animator** [comment]
Generated code for POST 6 is incorrect and not working.

**Comment by ashutosh7i** [comment]
> Generated code for POST 6 is incorrect and not working.

Yes upon my testing it failed too, this is because of path and content importing images, i have found a fix for the same, the image has to be parsed using ```file_get_contents($filename),``` function.

i am creating a modified PR over it,
Also if POST 6 is not working, POST 7 shouldn't work too, 

is POST 6 the only test Failing??

**Comment by animator** [comment]
You need to verify each and every generated code as mentioned in the guide https://github.com/foss42/apidash/discussions/80

**Pull request merged**